src/main.cpp

```diff
@@ -195,6 +210,21 @@ String getFormatData(unsigned int pm25, unsigned int pm10){
   return json;
 }
 
+ void getHumidityRead(){
+   humidity = am2320.readHumidity();
+   temperature = am2320.readTemperature();
+   Serial.println("-->[AM2320] Humidity: "+ String(humidity) + " % Temperature: " + String(temperature) + " °C");
+   humint = round (humidity);
+   tmpint = round (temperature);
+ }
+
+void humidityLoop(){
+//  if(v25.size()==0) {
```

Comment:
Why not when the 0 is reached?

Reply:
Not worked, only v25.size() >= stime copied from averageLoop()